### DISCUSSION TITLE
Erreur 403 HA

### DISCUSSION THREAD

**Discussion by Lokdi** [bokub/conso-api#65] [Aide]
Bonjour, 
J'ai installé Home assistant le 24 Oct (hier) et j'ai ensuite installé le module pour le linky, sauf que j'ai une erreur 403.
Avant de vous ecrire j'ai verifié que les informations de configuration etait correct en faisant un test sur le site :  https://conso.boris.sh/ est j'ai bien une réponse 200.

En décryptant l'erreur 403, j'aurai soit :

- mon ip de bloqué 
- une erreur générée par Cloudflare indiquant que l'accès au site a été bloqué 
- un possible blocage de sécurité/problème de configuration de cookies, 

J'ai bien compris les limites de call api mais je ne pense pas être dans ce cas ( test du Token uniquement sur HM ou le site de test ), 

En copie le screen de la configuration et voici les informations de l'erreur : 
Cloudflare Ray ID : 8d832d0ab9bf1546
IP bloquée : 89.80.21X.XX

Pouvez-vous m'aider ?

Merci  

![Capture d’écran 2024-10-25 180023](https://github.com/user-attachments/assets/1861d02c-6682-4937-853d-07d4a926537d)

**Comment by bokub** [comment]
Salut @Lokdi

En recherchant l'adresse IP que tu me donnes, j'en trouve une qui a été bloquée le 11 septembre

Ce qui m'étonne, c'est que tu arrives à récupérer des données depuis https://conso.boris.sh qui devrait logiquement t'indiquer aussi un blocage, vu que ça utilise la même API.

à tout hasard, n'aurais-tu pas une adresse IP différente de celle de ton serveur Home Assistant?

**Comment by bokub** [comment]
En tout cas, sur la fin de soirée du 23 octobre (donc avant que tu installes Home Assistant), j'ai 47 requêtes bloquées avec un PRM qui commence exactement comme celui indiqué dans ta capture (`0227`), c'est bizarre non ?

**Reply by Lokdi** [reply]
J'ai du utilisé les tests avant d'installer le ha, j'ai récupéré mon token il y a déjà quelques semaines.
J'utilise le HA sur mon raspberry en local ça vient de ça à votre avis ?

**Reply by bokub** [reply]
- Qu'appelles-tu "utiliser les tests" ?
- Il y a quelques semaines est-ce que ça peut correspondre au blocage initial du 11 septembre ?
- Tu n'a pas répondu à ma question plus haut: ton adresse IP est-elle différente de celle de ton serveur Home Assistant ?
